framework/src/outputs/TableOutput.C

```diff
@@ -34,6 +34,9 @@ InputParameters validParams<TableOutput>()
   InputParameters params = validParams<AdvancedOutput<FileOutput> >();
   params += AdvancedOutput<FileOutput>::enableOutputTypes("postprocessor scalar vector_postprocessor");
 
+  // Option for writing vector_postprocessor time file
+  params.addParam<bool>("time_data", false, "When true and VecptorPostprocessor data exists, write a csv file containing the timestep and time information.");
```

Comment:
Another option - yes, a little late - would be to have a `#` commented header with the time in it...